index.html

```diff
@@ -0,0 +1,1402 @@
+            </a>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-12 card border-0">
+          <a href="#">
+            <img class="card-img-top" src="./img/lebron-james-nba-los-angeles-lakers.webp" alt="lebron">
+          </a>
+          <div class="card-body">
+            <a href="#">
+              <h2 class="card-title">LeBron James Says 'Not Only Black People' Deal with Racism in the U.S.</h2>
+            </a>
+          </div>
+        </div>
+        <div class="col-lg-4 col-md-12">
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/nba-national-anthem.webp" alt="doc">
+              </a>
+              <a href="#">
+                <h2 class="card-title">
+                  Doc Rivers Says Taking a Knee Was Hardest Part of Lakers vs Clippers Game
+                </h2>
+              </a>
+            </div>
+          </div>
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/nba-national-anthem.webp" alt="nba">
+              </a>
+              <a href="#">
+                <h2 class="card-title">Charles Barkley Blasted For Supporting Players Who Stand For Anthem</h2>
+              </a>
+            </div>
+          </div>
+          <div class="row card border-0">
+            <div class="card-body d-inline-block">
+              <a class="float-left mr-2" href="#">
+                <img class="img-fluid" src="./img/lebron-james-kawhi-leonard-los-angeles-lakers.webp" alt="kawhi">
+              </a>
+              <a href="#">
+                <h2 class="card-title">
+                  NBA Schedule: Clippers vs. Lakers Start Time, TV Channel, Live Stream</h2>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row w-100 d-inline-flex border-top">
+        <div class="col-12 col-md-8 d-inline-block my-4">
+          <h3>SUBSCRIBE</h3>
+          <div>
+            <img class="float-left mr-5" height="200" src="./img/cover-july-24-2020.jpg" alt="cover">
+            <div class="pt-5">
+              <h4>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+              <p>All Access + Weekly Delivery.
+                Choose A Membership That's Perfect for You!</p>
+              <button class="btn-secondary" type="button">SUBSCRIBE</button>
+            </div>
+          </div>
+        </div>
+        <div class="col-12 col-md-4 my-4">
+          <h3>NEWSLETTER</h3>
+          <div class="d-inline-flex h-100">
+            <div class="mr-2">
+              <img width="200" height="200" src="./img/130387482-newsletter-subscription-button-on-mobile-phone-screen-business-marketing-concept-.jpg" alt="">
+            </div>
+            <div>
```

Comment:
- [ ] In the small screen size, it does not have an image.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89235930-9e570c80-d5c5-11ea-8ed3-eecdac6e06b2.png) |![image](https://user-images.githubusercontent.com/61479648/89235946-a616b100-d5c5-11ea-8de4-aae377413663.png) |

- [ ] IT is breaking at 1022 px, and overflowing

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/89236029-e413d500-d5c5-11ea-96fb-1e17aac7b32e.png) | ![image](https://user-images.githubusercontent.com/61479648/89236057-eece6a00-d5c5-11ea-98a1-329c1cc2d084.png) |